packages/compass-connections-navigation/src/constants.tsx

```diff
@@ -0,0 +1,21 @@
+import { spacing } from '@mongodb-js/compass-components';
+
+// TODO: Currently we show placeholder for every collection/database item in the list, but
+// do we want to / need to?
```

Comment:
This was a deliberate design decision to prevent collection list and the content below it from "jumping" when collections finished loading. Why we'd want to remove that?

Reply:
sorry, what did we want to revert here? or is it reverted already? @gribnoysup @kmruiz

Reply:
note: this file is moved from `compass-databases-navigation`

Reply:
I think nothing needs to be reverted, looking at what the code does, it still keeps the placeholders as expected.